### ISSUE TITLE
Add temperature offset

### ISSUE THREAD

**Issue by napalmcsr** [Roastero/Openroast#51]
For those of us with cool running roasters, it would be awesome to have a way of just setting an offset so when I download recipes, I wouldn't have to change every temperature.

**Comment by int3ll3ct** [comment]
Looking at the existing UI, where do you think such a setting belongs?  Describe the journey you'd expect to take as a user wanting to specify a 'constant offset' (let's not get too fancy here...)

**Comment by napalmcsr** [comment]
I can see 2 ways of doing it.
one of just putting a constant offset on the recipe creation screen and can update a recipe automatically, or
Put a text box to be under the roast button that is non-volatile (and also settable by command line argument 😊)
I would choose the second, just in case I went through a lot of roasters.

**Comment by int3ll3ct** [comment]
I agree that either an edit box near the roast button, or an edit box in the recipe browser window, would be the best places for this.  Thanks.
